entrypoint.sh

```diff
@@ -2,24 +2,24 @@
 
 # Set up auth if required
 auth=""
-if [[ $5 && $6 ]]; then
-    auth="--user=$5:$6"
+if [[ $6 && $7 ]]; then
+    auth="--user=$6:$7"
 fi
 
 # Check if we delete or not resources
 delete=""
-if [[ $7 == true ]]; then
+if [[ $8 == true ]]; then
   delete="--delete"
 fi
 
 # Check if there is a tenant id
 tenant=""
-if [[ $8 ]]; then
-  tenant="--tenant=$8"
+if [[ $9 ]]; then
+  tenant="--tenant=$9"
 fi
-
-if [[ $3 == 'namespace_files' ]]; then
-  /app/kestra namespace files update "$1" "$2" --server="$4" "$auth" $delete "$tenant"
+3
```

Comment:
Shouldn't it be `$3`?
I know nothing about how to build an action but 3 ... is 3 right?

Reply:
I think my finger slipped off :'(